### PR TITLE
444: Enables per compile output prune

### PR DESCRIPTION
**Pull request by ademariag**
Fixes issue #444 

## Proposed Changes

  - Enables per compile output prune

User can now explicitly setup a prune policy on a per compile output basis

### CODE REVIEW

**Comment by uberspot** [comment]
Could we add a test for this? :)

File: kapitan/inputs/jsonnet.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def _search_imports(cwd, imp):
         output_obj = json.loads(json_output)
 
         output = kwargs.get("output", "yaml")
-        prune = kwargs.get("prune", False)
+        prune = kwargs.get("prune_input", False)
```
Comment (uberspot) [inline kapitan/inputs/jsonnet.py]: Not sure if this will work. I have the impression `kwargs` here is referencing `--prune` from the command line.

**Comment by ademariag** [comment]
@uberspot please take another look

**Comment by ademariag** [comment]
@uberspot can you merge as I am not authorized...

**Pull request merged**